app/src/main/java/com/dillxn/tactilesynth/MenuFragment.java

```diff
@@ -33,10 +38,67 @@ public void onCreate(Bundle savedInstanceState) {
                              Bundle savedInstanceState) {
-        View view = inflater.inflate(R.layout.fragment_menu, container, false);
+        view = inflater.inflate(R.layout.fragment_menu, container, false);
         synthFrag = (SynthFragment) fragmentManager.findFragmentByTag("synthPrime");
 
         // Inflate the layout for this fragment
         return inflater.inflate(R.layout.fragment_menu, container, false);
     }
-}
\ No newline at end of file
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        // GRABBING MENU BUTTONS AND MENU LAYOUTS
+        Button settingsButton = (Button) view.findViewById(R.id.settingsBtn);
+        Button tuningButton = (Button) view.findViewById(R.id.tuningBtn);
+        Button recordingButton = (Button) view.findViewById(R.id.recordingBtn);
+        Button effectsButton = (Button) view.findViewById(R.id.effectsBtn);
+
+        LinearLayout settingsMenu = (LinearLayout) view.findViewById(R.id.settings_menu);
+        LinearLayout tuningMenu = (LinearLayout) view.findViewById(R.id.tuning_menu);
+        LinearLayout effectsMenu = (LinearLayout) view.findViewById(R.id.effects_menu);
+        LinearLayout recordingMenu = (LinearLayout) view.findViewById(R.id.recording_menu);
+
+        // SET UP LISTENERS FOR MENU BUTTONS
+        settingsButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                settingsMenu.setVisibility(View.VISIBLE);
+                tuningMenu.setVisibility(View.GONE);
+                effectsMenu.setVisibility(View.GONE);
+                recordingMenu.setVisibility(View.GONE);
+            }
+        });
+
+        tuningButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                settingsMenu.setVisibility(View.GONE);
+                tuningMenu.setVisibility(View.VISIBLE);
+                effectsMenu.setVisibility(View.GONE);
+                recordingMenu.setVisibility(View.GONE);
+            }
+        });
+
+        recordingButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                settingsMenu.setVisibility(View.GONE);
+                tuningMenu.setVisibility(View.GONE);
+                effectsMenu.setVisibility(View.GONE);
+                recordingMenu.setVisibility(View.VISIBLE);
+            }
+        });
+
+        effectsButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                settingsMenu.setVisibility(View.GONE);
+                tuningMenu.setVisibility(View.GONE);
+                effectsMenu.setVisibility(View.VISIBLE);
+                recordingMenu.setVisibility(View.GONE);
```

Comment:
This works for now, but it would be better to be able to have a variable number of menus, 
have an array 'menus' and have a method that turns off all menus in the array, instead of having this repeated code